decidim-results/app/views/decidim/results/results/_share.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="text-center">
```

Comment:
Is this the same partial everywhere? If so I'd move it to `core/app/views/application/_share.html.erb`